lib/matplotlib/axes/_axes.py

```diff
@@ -2873,7 +2873,8 @@ def xywhere(xs, ys, mask):
             plot_kw['markeredgewidth'] = capthick
         # For backwards-compat, allow explicit setting of
         # 'mew' or 'markeredgewidth' to over-ride capthick.
-        for key in ('markeredgewidth', 'mew', 'transform', 'alpha', 'zorder'):
+        for key in ('markeredgewidth', 'mew', 'transform', 'alpha',
+                    'zorder', 'rasterized'):
```

Comment:
you know... I only added this one tiny bit of logic back in the day for handling "mew" in case users already figured out how to adjust the cap thickness (it was never documented). I intended to deprecate that at some point, but forgot to do that. I didn't think it would ever be used for other things like this. Perhaps there are other keyword arguments that should be checked?